### ISSUE TITLE
supporting HEAD for any endpoint

### ISSUE THREAD

**Issue by tingzhao** [janelia-flyem/libdvid-cpp#3]
This will allow a client to quickly check whether some data exists or not.

**Comment by DocSavage** [comment]
Ting, could you clarify which endpoints need HEAD?  Since each data type can implement the API in various ways, supporting HEAD will have to happen for each data type implementation.  For example, HEAD for keyvalue type is just checking if they given key is stored, while HEAD for a labelvol has to check if *any* "label + block index" key is stored.  So I need to know which API calls you'll need to quickly check for data.  

Would it be the following endpoints in labelvol?
* sparsevol
* sparsevol-by-point
* sparsevol-coarse

Or just the first one?

**Comment by DocSavage** [comment]
Also, shouldn't this issue be on DVID and not libdvid-cpp?

**Comment by stephenplaza** [comment]
The issue should probably be on both repos.  I can add an endpoint that can just call coarse vol or something that will return quickly.  When you add the appropriate DVID endpoint, I can just call that instead.

**Comment by tingzhao** [comment]
Yes, the issue should be on DVID. I posted it in the wrong place.
